### PR TITLE
adjust hosting manual

### PR DESCRIPTION
**Pull request by madebyTimo**
Part of Gamify-IT/issues#328

### CODE REVIEW

File: install-manuals/all-services/README.md [hunk 1]
```diff
@@ -1,21 +1,20 @@
 # How to run all services
 
-All services are in a `docker compose` file.
+All services are available as docker containers. The preferred way to host is to use a `docker compose` file.
 
-## Prerequisite
+We provide a template in our [run-config](https://github.com/Gamify-IT/run-config) repository. There is also a [manual](https://github.com/Gamify-IT/run-config/hosting-template#setup) how to setup the hosting environment.
```
Comment (delvh) [inline install-manuals/all-services/README.md:5]: Could you perhaps do it the other way round and pack the setup into the docs and link here from the run-config?

File: install-manuals/all-services/README.md [hunk 2]
```diff
@@ -1,21 +1,20 @@
...
+#### Docker Desktop
 
-Clone `https://github.com/Gamify-IT/prototype`
+[Install Docker Desktop on windows](https://docs.docker.com/desktop/windows/install/)
 
-## Run project
+[Install Docker Desktop on linux](https://docs.docker.com/desktop/linux/install/)
 
-Run in the directory of the cloned project `docker-compose up --build`
+[Install Docker Desktop on macos](https://docs.docker.com/desktop/mac/install/)
```
Comment (delvh) [inline install-manuals/all-services/README.md:20]: 1. `[I`**n**`stall Docker]`
2. `#### For developers: Docker Desktop`
3. Erm… Why is the `prerequisite` header the last header?  That goes against any design I've ever heard of. **Everyone** expects something after it as well, why else would it be a **pre**requisite?

File: install-manuals/all-services/README.md (hunk 1 above)
Comment (madebyTimo) [reply install-manuals/all-services/README.md:5]: I don't know if this is better. Now the manual is at the same place as the files for which it is for.

File: install-manuals/all-services/README.md (hunk 2 above)
Comment (madebyTimo) [reply install-manuals/all-services/README.md:20]: 2. Why is Docker Desktop for developers? In my opinion the decision of using a GUI has nothing todo with that. 

3. I think docker is so fundamental that it don't has to be mentioned at the beginning. What about if we add a notice at the beginning and link the `Prerequisite` section there

Comment (delvh) [reply install-manuals/all-services/README.md:20]: 2. As I've recently heard: `You don't use Docker Desktop in Production. When you exit it, your deployments are killed as well…`.
3. I don't quite understand what you mean.

Comment (madebyTimo) [reply install-manuals/all-services/README.md:20]: 2. Ok I never heard that. What I know is that Docker ~~Desktop~~ is hard to install on windows (Needs a VM or some advanced setup in WSL). What about we add a notice "Not recommended for production use"?

Comment (delvh) [reply install-manuals/all-services/README.md:20]: Regarding 2: I think it was @robojones, who told me that.

**Comment by madebyTimo** [comment]
I forgot to mention it in the commit message but I also changed the order

File: install-manuals/all-services/README.md [hunk 3]
```diff
@@ -1,20 +1,24 @@
...
 
 [Install Docker Desktop on windows](https://docs.docker.com/desktop/windows/install/)
 
 [Install Docker Desktop on linux](https://docs.docker.com/desktop/linux/install/)
 
 [Install Docker Desktop on macos](https://docs.docker.com/desktop/mac/install/)
+
+## Run
+
+All services are available as docker containers. The preferred way to host is to use a `docker compose` file. Make sure you match the
```
Comment (delvh) [inline install-manuals/all-services/README.md:19]: you match the?

Comment (madebyTimo) [reply install-manuals/all-services/README.md]: Oh seem like I started a sentence that zou need to match the requirements but decided then to change the order

File: install-manuals/all-services/README.md [hunk 4]
```diff
@@ -1,21 +1,21 @@
...
+[Install Docker Desktop on macos](https://docs.docker.com/desktop/mac/install/)
 
-Clone `https://github.com/Gamify-IT/prototype`
+## Run
 
-## Run project
+All services are available as docker containers. The preferred way to host is to use a `docker compose` file.
 
-Run in the directory of the cloned project `docker-compose up --build`
+We provide a template in our [run-config](https://github.com/Gamify-IT/run-config) repository. There is also a [manual](https://github.com/Gamify-IT/run-config/hosting-template#setup) how to setup the hosting environment.
```
Comment (cdingm) [inline install-manuals/all-services/README.md:21]: the link to the manual is leading to 404

Comment (madebyTimo) [reply install-manuals/all-services/README.md:21]: Oh sorry its the wrong url. I will fix it

**Review by cdingm — APPROVED** [review]

**Review by delvh — APPROVED** [review]

**Pull request merged**